recipe/meta.yaml

```
@@ -23,30 +22,34 @@ build:
 
 requirements:
   host:
```

```suggestion
  build:
    - {{ compiler('c') }}
  host:
```
Because anaconda-linter complains:
`cython_needs_compiler: Cython generates C code, which will need to be compiled`
It's required if you enable `FONTTOOLS_WITH_CYTHON=1`

---

Added the compiler!